analyzers/src/SonarAnalyzer.Common/AnalysisContext/SonarAnalysisContextBase.cs

```diff
@@ -131,11 +133,34 @@ public bool HasMatchingScope(DiagnosticDescriptor descriptor)
             descriptor.CustomTags.Contains(tag);
     }
 
+    /// <summary>
+    /// Check if the current file path is included or excuded and caches the result.
```

Comment:
What is the current path? This comment creates more questions for me than it resolves.
Also if you are adding a summary for the method, it would be nice to also add the description for the parameter(s).

Reply:
Removed the `<summary>` since it doesn't provide value (this method isn't private only to allow its usage inside the Utility Analyzer in the follow up PR).